src/components/Element/ArticleElement/ArticleContext.jsx

```diff
@@ -6,10 +6,16 @@ export const useArticleContext = () => useContext(ArticleContext);
 export function ArticleProvider({ children }) {
   const [message, setMessage] = useState('');
   const [sentence, setSentence] = useState({});
+  const [wordList, setWordList] = useState([]);
 
```

Comment:
This is a lot of logic packed into one context/logic. Can you split this so not everything is coupled together.

Reply:
but it put together because here is a provider.

Reply:
This provider/context is handling a lot of logic, would it be easier to change/handle if we split the logic up?

Reply:
This provider is handling common data, maybe move to outside would easier to understand ?

Reply:
I remove function to tip